draft-ietf-satp-core.md

```diff
@@ -424,28 +438,36 @@ This is the hash of the current message payload.
 
 ### Signature Algorithms Supported
 
-This is the list of digital signature algorithm supported by a gateway, 
-with the base default being the NIST ECDSA signature algorithm with the P-256 curve and the SHA-256 hash function.
+This is a JSON list of digital signature algorithms supported by a
+gateway.  Each entry in the list should either an Algorithm Name value registered in the IANA "JSON Web Signature and Encryption Algorithms" registry established by {{JWA}} or be a value that contains a Collision-Resistant Name.
```

Comment:
2 spaces between `gateway.` and `Each`.
typo, missing "be" in `either an Algorithm`

otherwise lgtm